src/main/java/org/opensearch/security/ssl/OpenSearchSecuritySSLPlugin.java

```diff
@@ -318,15 +318,17 @@ public Collection<Object> createComponents(Client localClient, ClusterService cl
     @Override
     public List<Setting<?>> getSettings() {
         List<Setting<?>> settings = new ArrayList<Setting<?>>();
+
+        // add secure settings
+        SecureSSLSettings.addSettingsToBuilder(settings);
```

Comment:
I got confused the first time reading this. I see now that this method adds to the `settings` list that is passed into the method.